config/gnbsim.yaml

```diff
@@ -71,6 +76,7 @@ configuration:
       gnbName: gnb1 # gNB to be used for this profile
       startImsi: 208930100007497 # First IMSI. Subsequent values will be used if ueCount is more than 1
       ueCount: 5 # Number of UEs for for which the profile will be executed
+      defaultAs: "192.168.250.1" #default icmp pkt destination
```

Comment:
should we name this parameter as dataPktDst ??

Reply:
I was thinking like, default application Server (defaultAs)